diff --git a/CMake/FindSUNDIALS.cmake b/CMake/FindSUNDIALS.cmake
index 96c2de5..f08bbab 100644
--- a/CMake/FindSUNDIALS.cmake
+++ b/CMake/FindSUNDIALS.cmake
@@ -48,8 +48,7 @@ if(SUNDIALS_DIR)

     find_library(SUNDIALS_LIB
                  NAMES ${TEST_LIB}
-                 HINTS ${SUNDIALS_LIBRARY_DIR}
-                 NO_DEFAULT_PATH)
+                 HINTS ${SUNDIALS_LIBRARY_DIR})

     message(STATUS "SUNDIALS_LIB=${SUNDIALS_LIB}")
     if(SUNDIALS_LIB)
